test/tests/KeyboardPrefixSelectionMixin.tests.js

```diff
@@ -96,7 +113,9 @@ function createSampleElement() {
     'Blueberry',
     'Cantaloupe',
     'Cherry',
-    'Cranberry'
+    'Cranberry',
+    'E berry',
+    'E fruit',
```

Comment:
Not pretty, but https://simple.wikipedia.org/wiki/List_of_fruits didn't have anything with a common prefix followed by a space :)